policyengine_us/variables/gov/states/va/tax/income/subtractions/va_real_estate_investment_trust_subtraction.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_us.model_api import *
+
+
+class va_real_estate_investment_trust_subtraction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Virginia Real Estate Investment Trust subtraction"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://law.lis.virginia.gov/vacodefull/title58.1/chapter3/article2/",
+        "§ 58.1-322.02.(28.a.)",
```

Comment:
```suggestion
    reference = (
        title="§ 58.1-322.02.(28.a.)",
        href="https://law.lis.virginia.gov/vacodefull/title58.1/chapter3/article2/",
```
@PavelMakarchuk is it ok to write reference like this?

Reply:
yes that works well here